MEETINGS.md

```diff
@@ -6,6 +6,7 @@
 
 |  #  | Brand              | Type        | Name                        | Origin      | Tea Master      | Date     |
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------|----------|
+| 348 | [adagio teas]      | [Herbal]    | honeybush hazelnut          | [USA]       | Amy Dreise      | 07-05-18 |
```

Comment:
`[adagio teas]` - upper case
`honeybush hazelnut` - should be a link, upper case